src/App.tsx

```diff
@@ -16,35 +20,69 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.Default);
+  const [isReverse, setReverse] = useState(false);
+  const visibleGoods = sortGoods(goodsFromServer, sortField, isReverse);
+
   return (
     <div className="section content">
       <div className="buttons">
-        <button type="button" className="button is-info is-light">
+        <button
+          type="button"
+          className={classNames('button is-info', {
+            'is-light': sortField !== SortType.ALPHABET,
+          })}
+          onClick={() => {
+            setSortField(SortType.ALPHABET);
+          }}
+        >
           Sort alphabetically
         </button>
 
-        <button type="button" className="button is-success is-light">
+        <button
+          type="button"
+          className={classNames('button is-success', {
+            'is-light': sortField !== SortType.LENGTH,
```

Comment:
Consider adding a visual indicator to show which sorting option is currently active. This will enhance the user experience by making it clear which sorting method is applied.